glossary.html

```diff
@@ -1,10 +1,8 @@
 <!DOCTYPE html>
 <html lang = "en">
 <head>
-    <div class = "burger">
-    </div>
     <meta charset = "utf-8">
-    <meta name = "viewport" content = "width = device-width, initial-scale = 1.0">
+    <meta name = "viewport" content = "width=device-width, initial-scale=1.0">
```

Comment:
I haven't actually been able to find any information about whether spaces around the = signs here are allowed or not, but it's definitely the convention not to, so probably best to be safe here...